api-server/server/boot/certificate.js

```diff
@@ -496,9 +486,9 @@ function createShowCert(app) {
           messages: [
             {
               type: 'info',
-              message: dedent`
-              ${username} has not yet agreed to our Academic Honesty Pledge.
-            `
+              message: 'flash.msg-36',
```

Comment:
FYI - I noticed that `msg-36` has a typo in the `translations.json` fiile (see [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/b8ce82c4f8c6c34b5855ed5b3fcc47f2efd2d78a/client/i18n/locales/english/translations.json#L432).

`agrees` should be `agreed`
```js
"msg-36": "{{username}} has not yet agreed to our Academic Honesty Pledge.",
```